connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -990,6 +990,9 @@ message CollectMetrics {
 
   // (Required) The metric sequence.
   repeated Expression metrics = 3;
+
+  // (Required) A unique DataFrame id.
+  int64 dataframe_id = 4;
```

Comment:
since `dataframe_id` is set the `plan_id`

why not reusing the `plan_id` in `RelationCommon`?

Reply:
Good point. done